styles/Vocab/PingCAP/reject.txt

```diff
@@ -0,0 +1,9 @@
+TiBD
+(?i)a lot
+(?i)lots of
+(?i)many
+(?i)much
+(?i)enough
+(?i)sufficient
+(?i)a large number of
+(?i)a large volume of
```

Comment:
@TomShawn PTAL at the reject list. Any occurrence of these words will be flagged as an error.
If you find any other words that should be rejected, feel free to add them.

**Note**: `(?i)` means case insensitive.

Reply:
LGTM